lib/HLSL/DxilValidation.cpp

```diff
@@ -2134,17 +2134,39 @@ static void ValidateBarrierFlagArg(ValidationContext &ValCtx, CallInst *CI,
   }
 }
 
+std::string GetLaunchTypeStr(DXIL::NodeLaunchType LT) {
+  switch (LT) {
+  case DXIL::NodeLaunchType::Broadcasting:
+    return "Broadcasting";
+  case DXIL::NodeLaunchType::Coalescing:
+    return "Coalescing";
+  case DXIL::NodeLaunchType::Thread:
+    return "Thread";
+  case DXIL::NodeLaunchType::Invalid:
+  case DXIL::NodeLaunchType::LastEntry:
+    return "";
```

Comment:
Should this be "Invalid"?

Reply:
That would be because of a bad rendering of my last comment.  It was written as `"<invalid>"` but without the back-ticks around it, so when rendering the comment, it dropped what it thought was an HTML tag 😜.  Also, we shouldn't need the separate cases, we just need a single `default:` case.